### DISCUSSION TITLE
How to design the sending and receiving of arm system?

### DISCUSSION THREAD

**Discussion by dangbo08** [svpcom/wfb-ng#391] [Questions]
你好，请问一下，我想知道我的arm系统如果支持WiFi网卡和wfb的相关依赖库，是不是我只需要交叉编译出来的wfb_tx.wfb_rx,wfb_keygen这三个文件，就可以自行设计相关的接收和发送文件的方式？谢谢！！

**Comment by archfelix** [comment]
我也想知道如何使用wfb_xxx设计自己的收发，，而不是基于现有工具链，，你现在知道办了吗？
